reporters_db/data/reporters.json

```diff
@@ -870,6 +890,11 @@
                     "start": "1904-01-01T00:00:00"
                 }
             },
+            "examples": [
+                "11 Am. Ann. Cas. 198",
+                "Am.Ann.Cas. 1916C, 1093",
+                "Am.Ann.Cas. 1916E,157"
+            ],
```

Comment:
@quevon24 you added these cases but this isnt doing anything.  The standard is a full cite if no regex is set.  

Two of these are totally non functional - which makes me wonder if we have other situations where we've simply added examples but not implemented them.